libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -129,6 +129,10 @@ bool AC_Fence::pre_arm_check_polygon(const char* &fail_msg) const
         return false;
     }
 
+    if (!_poly_loader.check_inclusion_circle_margin(_margin)) {
+        fail_msg = "FENCE_MARGIN is less than inclusion circle radius";
```

Comment:
Can you check if this displays on a single line?  I think our limit is 54 characters and this would be 57 if "PreArm: " is prefixed to it.